components/mpas-ocean/src/shared/mpas_ocn_thick_ale.F

```diff
@@ -125,10 +125,12 @@ subroutine ocn_ALE_thickness(verticalMeshPool, SSH, ALE_thickness, &
          nCells        ! number of cells
 
       real (kind=RKIND) :: &
-         weightSum,    &! sum of weights in vertical
-         thicknessSum, &! total thickness
-         remainder,    &! track remainder in mix/max alteration
-         newThickness   ! temp used during min/max adjustment
+         weightSum,       &! sum of weights in vertical
+         restingThicknessSum,  &! total resting thickness
+         restingThicknessDiff, &! difference from total resting thickness
```

Comment:
Could we call `restingThicknessDiff` something else (like `restingSSH`)?  It isn't clear what it's the difference with respect to.